packages/peregrine/lib/talons/Gallery/gallery.gql.ce.js

```diff
@@ -0,0 +1,42 @@
+import { gql } from '@apollo/client';
+
+import { GET_PRODUCTS_IN_WISHLISTS } from '../Wishlist/GalleryButton/galleryButton.gql';
+
+export const GET_WISHLIST_CONFIG = gql`
+    query GetWishlistConfigForGalleryCE {
+        storeConfig {
+            id
+            magento_wishlist_general_is_enabled
+        }
+    }
+`;
+
+export const GET_WISHLIST_ITEMS = gql`
+    query GetWishlistItemsForLocalField($currentPage: Int!) {
+        customer {
+            id
+            wishlists {
+                id
+                items_v2(currentPage: $currentPage, pageSize: 10) {
```

Comment:
Do you think it would be cleaner to use `items` instead here? In an effort to make code look cleaner while using this in a talon?

Reply:
Since `items` is an actual field of `Wishlist`, I would avoid that alias, as not to cause confusion. When you see property references that don't follow our styleguide, it's also a good indicator you're working with GraphQL data. Developers can still alias on destructure, which we commonly do, eg. `const {items_v2: items} = data.wishlists`.